### ISSUE TITLE
Create a CSV report of newly ineligible users 

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/lib_jobs#700]
### User story
A daily CSV report of users who moved to an ineligible state.  Data elements to include are netID, name, and patron group. 

### Acceptance criteria

- [ ] Create a simple CSV file on these updated users on lib-jobs
- [ ] Make them available for download from somewhere behind CAS. This data falls under this [category](https://protectingourinfo.princeton.edu/unrestricted-within-princeton) for the University Info Security Policy. 

This is a follow-up issue to be done after #659 is completed.

**Comment by mzelesky** [comment]
This sounds familiar, but I think it's for users that become newly ineligible, so that we can validate that the ineligible users coming through the feed are indeed ineligible for library services.

**Comment by kevinreiss** [comment]
Ask Joan if this is still needed at next circ meeting.

**Comment by mzelesky** [comment]
Joan said this report is not needed.
